docs/about/features_index/taskrunner.rst

```diff
@@ -47,6 +47,7 @@ Configurable Settings
  - :code:`write_logs`: (boolean) Metric logging callback feature. By default, logging is done through `tensorboard <https://www.tensorflow.org/tensorboard/get_started>`_ but users can also use custom metric logging function for each task.     
  - :code:`persist_checkpoint`: (boolean) Specifies whether to enable the storage of a persistent checkpoint in non-volatile storage for recovery purposes. When enabled, the aggregator will restore its state to what it was prior to the restart, ensuring continuity after a restart. 
  - :code:`persistent_db_path`: (str:path) Defines the persisted database path. 
+ - :code:`save_native_model`: (boolean): Specifies whether to save the model in its native format. This is useful for post-experiment analysis and deployment. The model will be saved in the base path specified by the :code:`last_state_path` and :code:`best_state_path` setting.

```

Comment:
"Only applicable for tasks involving federated training, this is ignored for federated evaluation. If set, the model is saved in the framework native format at the end of the experiment. Saving methods are defined for each framework under `save_native` function of the respective [supported](https://github.com/securefederatedai/openfl/tree/develop/openfl/federated/task) framework plugins."

P.S.: Please check for no side-effects of this callback in evaluation mode.